code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -222,6 +226,9 @@
 			feedback_inc("cyborg_medical",1)
 
 		if("Security")
+			if(!config.allow_secborg)
+				src << "spam class = 'warning'>ERROR: NANOTRASEN HAS DISABLED THIS MODULE. PLEASE SELECT A DIFFERENT ONE!</span>"

```

Comment:
Why even have this section.
you can't pick it if it's not in the list.

this can't even be spoofed.